scripts/grest-helper-scripts/setup-grest.sh

```diff
@@ -94,25 +111,62 @@
   install_cron_job() {
     local job=$1
     local cron_pattern=$2
-    local cron_job_path="${CRON_DIR}/${job}"
-    if is_file "$CRON_DIR/${job}"; then
-      sudo rm "$CRON_DIR/${job}"
-    fi
-    local cron_job_entry="${cron_pattern} ${USER} /bin/sh ${CRON_SCRIPTS_DIR}/${job}.sh >> ${LOG_DIR}/${job}.log"
+    local cron_job_path="${CRON_DIR}/${CNODE_VNAME}-${job}"
+    local cron_scripts_path="${CRON_SCRIPTS_DIR}/${job}.sh"
+    local cron_log_path="${LOG_DIR}/${job}.log"
+    local cron_job_entry="${cron_pattern} ${USER} /bin/bash ${cron_scripts_path} >> ${cron_log_path}"
+    remove_cron_job "${job}"
     sudo bash -c "{ echo '${cron_job_entry}'; } > ${cron_job_path}"
   }
 
+  set_cron_variables() {
+    local job=$1
+    [[ ${PGDATABASE} != cexplorer ]] && sed -e "s@DB_NAME=.*@DB_NAME=${PGDATABASE}@" -i "${CRON_SCRIPTS_DIR}/${job}.sh"
+  }
+
+  set_cron_asset_registry_testnet() {
+    sed -e "s@CNODE_VNAME=.*@CNODE_VNAME=${CNODE_VNAME}@" \
+        -e "s@TR_URL=.*@TR_URL=https://github.com/input-output-hk/metadata-registry-testnet@" \
+        -e "s@TR_SUBDIR=.*@TR_SUBDIR=registry@" \
+        -i "${CRON_SCRIPTS_DIR}/asset-registry-update.sh"
+  }
+
   setup_cron_jobs() {
-    if ! is_dir "${CRON_SCRIPTS_DIR}"; then
-      mkdir "${CRON_SCRIPTS_DIR}"
-    fi
+    ! is_dir "${CRON_SCRIPTS_DIR}" && mkdir -p "${CRON_SCRIPTS_DIR}"
+
     get_cron_job_executable "stake-distribution-update"
+    set_cron_variables "stake-distribution-update"
     install_cron_job "stake-distribution-update" "*/30 * * * *"
     
     get_cron_job_executable "pool-history-cache-update"
+    set_cron_variables "pool-history-cache-update"
     install_cron_job "pool-history-cache-update" "*/10 * * * *"
+
+    if [[ ${NWMAGIC} -eq 764824073 || ${NWMAGIC} -eq 1097911063 ]]; then
```

Comment:
Comment -> Not applicable/Future addition for Guild network

Reply:
Couldn't guild network just be added later on?
As we don't have a token registry repository now, I thought it would be better just not deploy anything for that yet. Else, what should it do in the meantime? Should be an easy update later on once/if we deploy something.

Reply:
I think it was just a comment not to remember the guild network. We could do a mock token registry repo with the same structure, but I don't think we need to do anything for the guild network for this ATM.

Reply:
Ok, yeah I think it should be easy enough to add later on and have people run setup script again to deploy cron job for it.

Reply:
Yip just comment within script, to avoid someone noticing it later and thinking we missed it 👍🏻